admin/acceptance.py

```diff
@@ -91,6 +91,7 @@ def check_result(f):
             FLOCKER_ACCEPTANCE_CONTROL_NODE=control_node.address,
             FLOCKER_ACCEPTANCE_AGENT_NODES=':'.join(
                 node.address for node in agent_nodes),
+            FLOCKER_ACCEPTANCE_VOLUME_BACKEND='loopback',
```

Comment:
Note, this only informs the test what backend is being used, not which is running. The changes to systemd/upstart files mean that the loopback backend will be running.